web/src/Users.js

```diff
@@ -0,0 +1,8 @@
+import React, { Component } from 'react';
+import './App.css';
```

Comment:
This import is not used at the moment. Moreover, this component should probably be named `User`? It seems like it'll be used for single user.

Reply:
Same about all imports below - generally, `Users.js` should import only `Users.css`.